app/controllers/voucher_adjustments_controller.rb

```diff
@@ -5,6 +5,8 @@ class VoucherAdjustmentsController < BaseController
 
   def create
     if add_voucher
+      VoucherAdjustmentsService.calculate(@order)
```

Comment:
This seems to be a fixup for the previous commit.

Reply:
I think this commit changed a bit after a rebase. I'll squash it :ok_hand: